src/errors.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+// This is the error format used to report errors to the caller-supplied
+// error-handler
+export class NativeError extends Error {
```

Comment:
The fact that its data lives in the context of prepack/the caller rather than the context of the program being run.
I've changed it to ProgramEvaluationError, though I'm not sure if that's much better... let me know if you have other suggestions